src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/config/ResourceServerConfiguration.kt

```diff
@@ -32,18 +34,16 @@ class ResourceServerConfiguration(
           customizer.anyRequestRoleCustomizer.defaultRole
             ?.also { authorize(anyRequest, hasRole(it)) }
             ?: also {
-              // TODO: This has to be a user role - not ideal but will secure with this for now
-              authorize("/report/**", hasRole(dprEndpointApiRole))
-              authorize("/reports/**", hasRole(dprEndpointApiRole))
-              authorize("/definitions/**", hasRole(dprEndpointApiRole))
-              authorize("/statements/**", hasRole(dprEndpointApiRole))
+              authorize("/report/**", hasAnyRole(*dprEndpointApiRoles.toTypedArray()))
+              authorize("/reports/**", hasAnyRole(*dprEndpointApiRoles.toTypedArray()))
+              authorize("/definitions/**", hasAnyRole(*dprEndpointApiRoles.toTypedArray()))
+              authorize("/statements/**", hasAnyRole(*dprEndpointApiRoles.toTypedArray()))
               authorize(anyRequest, authenticated)
             }
         }
     }
     oauth2ResourceServer {
-      // TODO: will allow override of the DprAuthAwareTokenConverter
-      jwt { jwtAuthenticationConverter = AuthAwareTokenConverter() }
+      jwt { jwtAuthenticationConverter = DefaultDprAuthAwareTokenConverter(caseloadProvider) }
```

Comment:
This is the magic line I couldn't add before without the DPR library change yesterday